From: Scott Leggett <scott@sl.id.au>
Date: Tue, 26 Feb 2013 15:19:22 +1100
Subject: Terrible hack to build in cowbuilder environment.

---
 Makefile |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 13a9227..ec3cf04 100644
--- a/Makefile
+++ b/Makefile
@@ -29,8 +29,6 @@ endif
 # ==== PCRE regular expression ====
 ifeq "$(NO_REGEX_PCRE)" ""
   CFG += -DCONFIG_REGEX_PCRE
-  LIBS += $(shell pcre-config --libs)
-  INCS += $(shell pcre-config --cflags)
   ifeq "$(NO_REGEX_PCRE_JIT)" ""
     CFG += -DCONFIG_REGEX_PCRE_JIT
   endif
@@ -45,7 +43,6 @@ endif
 ifeq "$(NO_VSYNC_OPENGL)" ""
   CFG += -DCONFIG_VSYNC_OPENGL
   # -lGL must precede some other libraries, or it segfaults on FreeBSD (#74)
-  LIBS := -lGL $(LIBS)
 endif
 
 # ==== D-Bus ====
@@ -68,8 +65,6 @@ CFG += -DCOMPTON_VERSION="\"$(COMPTON_VERSION)\""
 LDFLAGS ?= -Wl,-O1 -Wl,--as-needed
 CFLAGS ?= -DNDEBUG -O2 -D_FORTIFY_SOURCE=2
 
-LIBS += $(shell pkg-config --libs $(PACKAGES))
-INCS += $(shell pkg-config --cflags $(PACKAGES))
 
 CFLAGS += -Wall
 
@@ -87,6 +82,11 @@ src/.clang_complete: Makefile
 	$(CC) $(CFG) $(CFLAGS) $(INCS) -c src/$*.c
 
 compton: $(OBJS)
+	LIBS += $(shell pcre-config --libs)
+	INCS += $(shell pcre-config --cflags)
+	LIBS := -lGL $(LIBS)
+	LIBS += $(shell pkg-config --libs $(PACKAGES))
+	INCS += $(shell pkg-config --cflags $(PACKAGES))
 	$(CC) $(CFG) $(LDFLAGS) $(CFLAGS) -o $@ $(OBJS) $(LIBS)
 
 man/%.1: man/%.1.asciidoc
